pkg/cmd/release/create/create.go

```diff
@@ -658,9 +680,13 @@ func printPackageVersions(ioWriter io.Writer, packages []*StepPackageVersion) er
 	t.AddRow(output.Dim("PACKAGE"), output.Dim("VERSION"), output.Dim("STEPS"))
 
 	for _, pkg := range consolidated {
+		version := pkg.Version
+		if version == "" {
+			version = output.Red("ERROR") // can't determine version for this package
```

Comment:
It looks like this

![image](https://user-images.githubusercontent.com/6362/186143501-c4364f0d-108b-4e38-b8f0-6d585ee71ba7.png)